Felienne/hedy
# Added pre-commit hook for prettier

**Description**
Added pre-commit configuration to run Prettier when committing code. This should make code-reviews much easier and keep styling consistent across the repository.

**Fixes _issue or discussion number_**
No linked issue, was discussed on Discord

**How to test**
Since prettier has run on many many files, the changes are too big to code-review. I suggest you reproduce the following steps on the main branch instead, and then merge my pull request into it to verify I haven't changed anything else than the formatting.

1. Install pre-commit using pip
2. Add pre-commit to `requirements.txt`
3. Add `pre-commit-config.yaml` with the same values as in my branch
4. Run `pre-commit install` to install the prettier package for pre-commit
5. Run `pre-commit run --all-files` to run prettier across all files in the directory, instead of just when changes are made to a file. This is a one-time operation
